splink/comparison_level_library.py

```diff
@@ -514,12 +509,18 @@ def create_label_for_charts(self) -> str:
         )
 
 
+DateMetricType = Literal["second", "minute", "hour", "day", "month", "year"]
```

Comment:
this results in the metric having an autocomplete list in vs code like if it were an enum